app/(tabs)/_layout.tsx

```diff
@@ -0,0 +1,38 @@
+import { Tabs } from 'expo-router';
+import { TabBarIcon } from '../../components/TabBarIcon';
+import { colors } from '../../constants/colors';
+
+export default function TabLayout() {
+  return (
+    <Tabs
```

Comment:
As mentioned in the PR description, let's decide whether we want to stay with expo-router bottom tabs or whether we want to use this new (native) option: 

- https://github.com/upleveled/expo-example-spring-2024-atvie/issues/9

Reply:
As @karlhorky and I were talking offline, we would stick to using the Expo Tabs with Expo Vector Icons 
https://docs.expo.dev/router/advanced/tabs/